Dockerfile

```diff
@@ -21,6 +13,12 @@ RUN if [ "$INSTALL_CPU_PROFILER" = "true" ]; then \
         rm -rf /pyflame; \
     fi
 
+# Make a directory for private credentials files
+RUN mkdir /auth
```

Comment:
I'd prefer this to be 'credentials' rather than 'auth', auth is usually authenticating users of a service, rather than authenticating the serivce to something else.

Reply:
changed